Spring_Server/gs-uploading-files-complete/target/classes/META-INF/MANIFEST.MF

```diff
@@ -1,9 +1,9 @@
 Manifest-Version: 1.0
 Implementation-Title: gs-uploading-files
 Implementation-Version: 0.1.0
-Built-By: localmgr
+Built-By: thelenzr
```

Comment:
Will this interfere with other users attempting to use this manifest to build the jar?

Reply:
Or is it just a tag noting the latest user who ran a `build`?